tests/cypress/latest/e2e/capa_kubeadm_clusterclass.spec.ts

```diff
@@ -1,17 +1,14 @@
 import '~/support/commands';
-import * as cypressLib from '@rancher-ecp-qa/cypress-library';
+import * as randomstring from "randomstring";
 import { qase } from 'cypress-qase-reporter/dist/mocha';
 import { skipClusterDeletion } from '~/support/utils';
 
 Cypress.config();
 describe('Import CAPA Kubeadm Class-Cluster', { tags: '@full' }, () => {
-  let clusterName: string
+  const separator = '-'
   const timeout = 1200000
-  const className = 'aws-kubeadm-example'
-  const repoName = 'class-clusters-aws-kb'
-  const branch = 'main'
-  const path = '/tests/assets/rancher-turtles-fleet-example/capa/kubeadm/class-clusters'
-  const repoUrl = 'https://github.com/rancher/rancher-turtles-e2e.git'
+  const classNamePrefix = 'aws-kubeadm'
+  const clusterName = 'turtles-qa'.concat(separator, classNamePrefix, separator, randomstring.generate({ length: 4, capitalization: 'lowercase' }), separator, Cypress.env('cluster_name_suffix'))
```

Comment:
My suggestion was to avoid using random string, and simply rely on cluster_name_suffix, that way names are more predictable.
```suggestion
  const clusterName = 'turtles-qa'.concat(separator, classNamePrefix, separator, Cypress.env('cluster_name_suffix'))
```

Reply:
Not sure it will solve problem for AKS/EKS/GKE runs. For eg. cluster names would not be unique `turtles-qa-aws-rke2-gh-runner`, and would require cleanup in case failed runs. This way also reduces manual input from user for suffix.

Reply:
I think having a more predictable cluster name **might** help when we want to write upgrade tests and do scaling operations post upgrade, assuming these upgrade and scaling operations are done in a file separate from the suite tests. I am still not sure how these tests would be structured so it might turn out that using these predictable names is not of much use.

Reply:
>So predictable cluster names for manual runs or nightly ?

Manual runs.